nixos/modules/services/networking/firewall.nix

```diff
@@ -42,6 +42,18 @@ let
 
   kernelHasRPFilter = ((kernel.config.isEnabled or (x: false)) "IP_NF_MATCH_RPFILTER") || (kernel.features.netfilterRPFilter or false);
 
+  targetChains = {
+    ACCEPT = "nixos-fw-accept";
+    DROP = "nixos-fw-refuse";
+    REJECT = "nixos-fw-log-refuse";
+    LOG = "LOG";
+    INPUT = "nixos-fw-input";
+    OUTPUT = "nixos-fw-output";
+    FORWARD = "nixos-fw-forward";
+    PREROUTING = "PREROUTING";
+    POSTROUTING = "POSTROUTING";
+  };
```

Comment:
It would be great if you could document these new rules at the beginning of the file.